src/data/misc/marketing.rs

```diff
@@ -0,0 +1,67 @@
+use rand::Rng;
+use wasm_bindgen::prelude::*;
+
+//These words can probably be improved?
+
+#[wasm_bindgen]
+pub fn buzzword() -> String {
```

Comment:
we should probably prefix this function with marketing so its `marketing_buzzword`,  otherwise LGTM.